src/package.json

```diff
@@ -37,6 +37,7 @@
     "http-errors": "1.6.3",
     "image-size": "0.9.7",
     "jade": "1.11.0",
+    "json-2-csv": "^4.0.0",
```

Comment:
@rekhaLuezoid please remove @json2csv/node from the list as it is not being used now.. Also, remove the '^' from the version. It should be  "json-2-csv": "4.0.0",